### ISSUE TITLE
Since this repo unmaintained about a year, I've done some efforts to make a new project `react-google-map-api`, maintainers are welcome for tests and examples. Link to the public repo inside.

### ISSUE THREAD

**Issue by JustFly1984** [tomchentw/react-google-maps#949]
https://github.com/JustFly1984/react-google-maps-api

Main issues resolved: 
  
- Simplified API and examples
-   Supports latest React, ready for async React.
-   Can render multiple maps per page.
-   Cleanup on unmount
-   New React context API
- Does not includes lodash or any accidental dependencies. Small bundle size (~29k)

Gonna publish it to npm till the end of the week.

**Comment by nikola1970** [comment]
Cool!
Will there be documentation and live examples? Thanks!

**Comment by AndyOGo** [comment]
Sounds awesome, thank you.
Lodash is exactly one of my problems...

**Comment by uriklar** [comment]
@nikola1970 we have some live examples and we've started working on docs. We'd love any help we can get! Adding docs is very simple. They're generated from mdx files that you can easily create by following existing examples.

All examples require you to create a google maps api key though..
Examples are currently runnable locally, but we'll get everything live soon enough

**Comment by JustFly1984** [comment]
Yes, since google pull up their prices we just can't use some default google api key.
You can clone a repo and run gatsby example website to see examples, and soon we will have docz on github pages.

**Comment by sibelius** [comment]
@tomchentw can you give @JustFly1984 and more developers access to this repo, so we can bring efforts together?

**Comment by JustFly1984** [comment]
I've published the lib https://www.npmjs.com/package/react-google-maps-api

**Comment by JustFly1984** [comment]
docs are at https://react-google-maps-api.netlify.com/

**Comment by JustFly1984** [comment]
Maintainers and contributors are welcome!

**Comment by zamarajev** [comment]
> docs are at https://react-google-maps-api.netlify.com/

Not Found

**Comment by firestar300** [comment]
@JustFly1984 ⬆️

**Comment by uriklar** [comment]
Here's the working link: https://react-google-maps-api-docs.netlify.com/

**Comment by zamarajev** [comment]
@uriklar Thanks!
